apps/condo/domains/ticket/constants/errors.js

```diff
@@ -1,5 +1,36 @@
 const UNKNOWN_GROUP_BY_FILTER = 'UNKNOWN_GROUP_BY_FILTER'
+const INVALID_PHONE_NUMBER_FORMAT = 'INVALID_PHONE_NUMBER_FORMAT'
+const NEGATIVE_TALK_TIME = 'NEGATIVE_TALK_TIME'
+
+const INVALID_PHONE_NUMBER_FORMAT_MESSAGE = 'Invalid phone number format'
```

Comment:
It’s better to use INVALID_CALLER_PHONE_NUMBER_FORMAT.message and hardcode this value